fdbserver/GrvProxyServer.actor.cpp

```diff
@@ -320,6 +326,7 @@ ACTOR Future<Void> getRate(UID myID,
 			batchTransactionRateInfo->setRate(rep.batchTransactionRate);
 			stats->transactionRateAllowed = rep.transactionRate;
 			stats->batchTransactionRateAllowed = rep.batchTransactionRate;
+			++stats->updatesFromRatekeeper;
```

Comment:
This function (`getRate`) should be updated with a little documentation at the top to describe what it does

Reply:
Done.